packages/bot-web-ui/src/components/quick-strategy/inputs/qs-input.tsx

```diff
@@ -15,14 +15,17 @@ type TQSInput = {
 };
 
 const QSInput: React.FC<TQSInput> = observer(
-    ({ name, onChange, type = 'text', fullwidth = false, attached = false, disabled = false }) => {
+    ({ name, type = 'text', fullwidth = false, attached = false, disabled = false }) => {
```

Comment:
@maryia-matskevich-deriv, is there any specific reason for removing the `onChange` from props here? I think from `form.tsx` this prop is still passed to `qs-input`, and a handler function is asynchronously handling the field value and field touch. Can you please have a look into this?

Reply:
taken back, thank you🙂